frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -110,16 +114,41 @@
 import ButtonCountdown, { countdownStates } from '../ButtonCountdown'
 import Sign from './Sign.vue'
 import L from '../../utils/translations'
-import { votingType } from '../../utils/validators'
+import { votingType, votesObj } from './validators'
 import template from 'string-template'
+import contracts from '../../../model/contracts.js'
+const { TypeInvitation, TypeRemoval, TypeIncome, TypeChangeThreshold, TypeApprovalThreshold, TypeRemovalThreshold } = contracts.GroupProposal
 
 export default {
   name: 'Voting',
   props: {
     type: {
-      validator: votingType
+      validator: votingType,
+      required: true
     },
-    proposal: Object,
+    votes: {
+      validator: votesObj,
+      required: true
+    },
+    value: {
+      type: [String, Number], // string for add member, remove member, number for rule, mincome
+      required: true
+    },
+    originalValue: Number, // for rule, mincome proposals
+    ownVote: [Boolean, null], // for (true), against (false) or not voted yet (null)
```

Comment:
`ownVote` is the wrong variable name to use for such a concept.

What is being represented here, according to the comment on the right, is an `enum` with 3 values, and it's not about whether this is your vote or not, it's about the state of the vote itself.

It is confused also with the following properties:

- type
- votes
- value
- originalValue
- isOwnProposal
- initiator

So all of these props need to be rewritten and simplified.